src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -385,9 +389,14 @@ public RealJenkinsRule includeTestClasspathPlugins(boolean includeTestClasspathP
                     }
                     System.out.println("Will load plugins: " + Stream.of(plugins.list()).filter(n -> n.matches(".+[.][hj]p[il]")).sorted().collect(Collectors.joining(" ")));
                     base.evaluate();
+                } catch (Throwable t) {
+                    LOGGER.log(Level.WARNING, "Something went wrong", t);
+                    throw t;
```

Comment:
I do not understand the purpose of this. If an error is thrown out of a test, the test fails and the error is displayed by the test runner right?

```suggestion
```

Reply:
I lost a lot of time understanding root cause with empty port because first exception was lost when finally throw it’s own exception, I still suggest leave it to at least log an exception.

Reply:
> I lost a lot of time understanding root cause with empty port because first exception was lost when finally throw it’s own exception

Sounds like a bug somewhere, the furst exception should be a `supressed` exception of whatever was thrown in the `finally` 

Perhaps a bug in junit runner or surefire or somewhere else where it is ignoring supressed exceptions (I expect there could be multiple cases of this).

https://issues.apache.org/jira/browse/SUREFIRE-1772  possibly.